src/consensus/votor.rs

```diff
@@ -252,12 +253,16 @@ impl<A: All2All + Sync + Send + 'static> Votor<A> {
         );
         let sender = self.event_sender.clone();
         tokio::spawn(async move {
-            tokio::time::sleep(DELTA_TIMEOUT).await;
+            tokio::time::sleep(DELTA_TIMEOUT + DELTA_FIRST_SLICE).await;
```

Comment:
I guess we need to create a similar issue in agave to handle this scenario.  I think it is still doing it the old way.

Reply:
We might need a new issue.  https://github.com/anza-xyz/alpenglow/blob/master/votor/src/timer_manager/timers.rs#L61.  We are generating the crashed leader timeout after DELTA_BLOCK.

Reply:
immediately after DELTA_TIMEOUT, before the first DELTA_BLOCK, if I am reading that correctly, but yes, either would warrant a new issue